snomed-ct-integration/src/test/java/dev/ikm/tinkar/snomedct/integration/DescriptionSemanticIT.java

```diff
@@ -65,4 +85,43 @@ public void testSemantic() {
 
         assertEquals(expectedFqn, actualFqn);
     }
+
+    /**
+     * Test FQN Term value.
+     * @result Term is validated agains a valid DataSet,
+     *
+     */
+    @Test
+    public void testFQN() {
+        // Given
+        String expectedTermFqn = "Erythema gyratum repens (disorder)";
+        String actualTermFqn = "";
+        String actualUUID = "ba8a3539-31fa-307a-895c-37401f0aea78"; //SNOMED CT integer ID: 77300003
+
+        UUID diseaseId = UUID.fromString(actualUUID);
+        Entity<EntityVersion> cldEntity = EntityService.get().getEntityFast(diseaseId);
```

Comment:
As a small nit, the "cld" in cldEntity stands of "chronic lung disease", but the test no longer refers to that term, so you can just rename this to "entity".